.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#pycharm
+.idea/
```

Comment:
you might want to add more to .gitignore using this https://www.toptal.com/developers/gitignore

Reply:
According to https://www.toptal.com/developers/gitignore/api/pycharm it's _.idea_ folder, which should be added to _.gitignore_ for PyCharm. All the recommended others folders/files were already added to _.gitignore_.